styles/stylesheets/screen.css

```diff
@@ -300,5 +232,5 @@ svg#graphics {
 }
 /* line 10, ../sass/partials/_table.scss */
 .dataTables_length label {
-  width: auto;
+  width: auto !important;
```

Comment:
pls add a comment saying that why this important

Reply:
I have added comment for this